src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Services/MockOuterApi.cs

```diff
@@ -1,25 +1,46 @@
 ﻿using System;
-using WireMock.RequestBuilders;
-using WireMock.ResponseBuilders;
 using WireMock.Server;
 
 namespace SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Services
 {
     public class MockOuterApi : IDisposable
     {
+        private bool _isDisposed;
+
         public WireMockServer MockServer { get; }
-        public Uri BaseAddress { get; }
+        public string BaseAddress { get; }
```

Comment:
Any particular reason this changed to a `string`?

Reply:
Mock Server returns it as a string and our tests require it as a string for the config values. Therefore, there is no value in holding it as anything other than a string

Reply:
I would argue that there are number of benefits - 
* documenting that this property contains a URI and not just any old collection of characters.  
* Enforcing the `Uri` invariants, hence ensuring that the `BaseAddress` could never contain a value that isn't a real URI.  
* Failing fast when configured with invalid URLs

`strings` (and `ints`, `longs`, etc.) are rarely a good type for data, they are often used because it is easier than creating a proper type for the concept being expressed.  In this case we don't need to create a type - the framework provides one for us.  I don't see any reason not to use it.